### ISSUE TITLE
The spin button for specifying the area of the property

### ISSUE THREAD

**Issue by mohammad-suliman** [hasadna/datacity-businessgate#9]
## Problem description:
The spin button (input field) for specifying the area of the property in meter squared has the following accessibility issues:
* It has no proper label. There is only a placeholder - a problematic technique from an accessibility perspective as discussed before
* The "submit" button has no label for assistive technologies
## Solution:
* Specify a label using accessibility approved technique
* Add a label for the "submit" button maybe with aria-label
